src/views/CanvasOrderbook/OrderbookControls.tsx

```diff
@@ -0,0 +1,140 @@
+import { useCallback } from 'react';
+
+import { clamp } from 'lodash';
+import { shallowEqual } from 'react-redux';
+import styled from 'styled-components';
+
+import { MarketOrderbookGrouping, Nullable, OrderbookGrouping } from '@/constants/abacus';
+import { ButtonShape, ButtonSize } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { Button } from '@/components/Button';
+import { Output, OutputType } from '@/components/Output';
+import { ToggleGroup } from '@/components/ToggleGroup';
+
+import { useAppSelector } from '@/state/appTypes';
+import { getCurrentMarketConfig } from '@/state/perpetualsSelectors';
+
+import abacusStateManager from '@/lib/abacus';
+
+type OrderbookControlsProps = {
+  className?: string;
+  assetName?: string;
+  selectedUnit: 'fiat' | 'asset';
+  setSelectedUnit(val: 'fiat' | 'asset'): void;
+  grouping: Nullable<MarketOrderbookGrouping>;
+};
+
+export const OrderbookControls = ({
+  className,
+  assetName,
+  selectedUnit,
+  setSelectedUnit,
+  grouping,
+}: OrderbookControlsProps) => {
+  const stringGetter = useStringGetter();
+  const modifyScale = useCallback(
+    (direction: number) => {
+      const start = grouping?.multiplier.ordinal ?? 0;
+      const end = clamp(start + direction, 0, 3);
+      abacusStateManager.modifyOrderbookLevel(
+        OrderbookGrouping.values().find((v) => v.ordinal === end)!
+      );
+    },
+    [grouping?.multiplier.ordinal]
+  );
+  const currentMarketConfig = useAppSelector(getCurrentMarketConfig, shallowEqual);
+  const { tickSizeDecimals: tSD } = currentMarketConfig ?? {};
+  const tickSizeDecimals = tSD ?? 2;
+
+  return (
+    <$OrderbookControlsContainer className={className}>
+      <$OrderbookUnitControl>
+        <$OrderbookLabel>{stringGetter({ key: STRING_KEYS.ORDERBOOK_UNITS })}</$OrderbookLabel>
+        <ToggleGroup
+          items={[
+            { label: assetName ?? '', value: 'asset' as const },
+            { label: 'USD', value: 'fiat' as const },
+          ]}
+          shape={ButtonShape.Rectangle}
+          value={selectedUnit}
+          onValueChange={setSelectedUnit}
+        />
+      </$OrderbookUnitControl>
+      <$OrderbookZoomControl>
+        <$OrderbookLabel>Grouping</$OrderbookLabel>
+        <$ZoomControls>
+          <Output
+            value={grouping?.tickSize}
+            type={OutputType.Fiat}
+            fractionDigits={tickSizeDecimals === 1 ? 2 : tickSizeDecimals}
+          />
+          <$ButtonGroup>
+            <Button
+              size={ButtonSize.XSmall}
+              shape={ButtonShape.Square}
+              onClick={() => modifyScale(-1)}
+            >
+              -
```

Comment:
this is a crazy small nit but the '-' looks slightly lower than '+' T__T any chance we can make them align?
<img width="220" alt="Screenshot 2024-06-07 at 4 12 49 PM" src="https://github.com/dydxprotocol/v4-web/assets/9400120/9fe54183-ebd2-438d-beb3-6c41b1c24613">

Reply:
lol good call, fixed